sdk/eventhub/event-hubs/src/eventData.ts

```diff
@@ -169,24 +166,7 @@ export namespace EventDataInternal {
         (data.properties as any)[prop] = (msg as any)[prop];
       }
     }
-    // Since rhea expects message headers as top level properties we will look for them and unflatten them inside header.
-    for (const prop of messageHeader) {
-      if ((msg as any)[prop] != undefined) {
-        if (!data.header) {
-          data.header = {};
-        }
-        (data.header as any)[prop] = (msg as any)[prop];
-      }
-    }
-    if (msg.application_properties) {
-      data.applicationProperties = msg.application_properties;
-    }
-    if (msg.delivery_annotations) {
-      data.lastEnqueuedOffset = msg.delivery_annotations.last_enqueued_offset;
-      data.lastSequenceNumber = msg.delivery_annotations.last_enqueued_sequence_number;
-      data.lastEnqueuedTime = new Date(msg.delivery_annotations.last_enqueued_time_utc as number);
-      data.retrievalTime = new Date(msg.delivery_annotations.runtime_info_retrieval_time_utc as number);
```

Comment:
I believe we need this in order to implement the `enableReceiverRuntimeMetric` feature

Reply:
Yes, we need this if `receiverRuntimeMetricEnabled` is enabled.